tests/snapshots/local/gc/gc-i31.wat.print

```diff
@@ -0,0 +1,7 @@
+(module
+  (type (;0;) (func (result i32)))
+  (func $f (;0;) (type 0) (result i32)
+    (local $a (ref i31)) (local $b (ref null i31)) (local $c (ref null i31))
```

Comment:
Could a special-case be added to `wasmprinter` to pring `(ref null i31)` as `i31ref` if that's an abbreviation?

Reply:
Seconding that having `wasmprinter` use the abbreviation short hand when applicable would be great, but it is fine if that is a follow up PR.

Reply:
Is there a good reason for that? (beside it being a historical convention)

`(ref null i31)` looks cleaner to me. It kinda bugs me that `i31ref` and `(ref i31)` are subtly different in this way. 

So, I personally would very much prefer `(ref null i31)` which is explicit and obvious.

Also, having to spend extra characters on `null` can makes one think twice if they really want to make a billion dollar mistake 🙂